src/fmripost_aroma/workflows/base.py

```diff
@@ -427,30 +476,42 @@ def init_single_run_wf(bold_file):
 
         bold_MNI6_wf = init_bold_volumetric_resample_wf(
             metadata=bold_metadata,
-            fieldmap_id=None,  # XXX: Ignoring the field map for now
+            fieldmap_id=fmapid,
             omp_nthreads=omp_nthreads,
             mem_gb=mem_gb,
             jacobian='fmap-jacobian' not in config.workflow.ignore,
             name='bold_MNI6_wf',
         )
-        bold_MNI6_wf.inputs.inputnode.motion_xfm = functional_cache['bold_hmc']
-        bold_MNI6_wf.inputs.inputnode.boldref2fmap_xfm = functional_cache['boldref2fmap']
-        bold_MNI6_wf.inputs.inputnode.boldref2anat_xfm = functional_cache['boldref2anat']
-        bold_MNI6_wf.inputs.inputnode.anat2std_xfm = functional_cache['anat2mni152nlin6asym']
         bold_MNI6_wf.inputs.inputnode.resolution = '02'
-        # use mask as boldref?
-        bold_MNI6_wf.inputs.inputnode.bold_ref_file = functional_cache['bold_mask_native']
         bold_MNI6_wf.inputs.inputnode.target_mask = mni6_mask
         bold_MNI6_wf.inputs.inputnode.target_ref_file = mni6_mask
 
+        # Warp desc-preproc_fieldmap to boldref space
+        # Warp the mask as well
+        fieldmap_to_boldref = pe.Node(
+            ApplyTransforms(),
+            name='fieldmap_to_boldref',
+        )
```

Comment:
This node needs to go into the `bold_MNI6_wf` because you want to resample from fmap to the MNI space.